libs/SalesforceReact/SalesforceReact/Classes/SFSmartSyncReactBridge.m

```diff
@@ -173,6 +174,15 @@ - (BOOL)isGlobal:(NSDictionary *)args
     return args[kSyncIsGlobalStoreArg] != nil && [args[kSyncIsGlobalStoreArg] boolValue];
 }
 
+- (void)handleCleanReSyncGhosts:(SFSyncStateStatus)syncStatus callback:(RCTResponseSenderBlock)callback
+{
+    if (syncStatus == SFSyncStateStatusDone) {
+        callback(@[[NSNull null], @"OK"]);
```

Comment:
Returning ok instead of done.
Also calling error callback on failure.
Will update the Android side in a future PR (and also will revisit the hybrid code).